src/main/java/org/broadinstitute/consent/http/models/AuthUser.java

```diff
@@ -20,7 +22,7 @@ public AuthUser(String email) {
   }
 
   public AuthUser deepCopy() {
```

Comment:
Why is this method needed? It looks like it's only used in `getUserWithStatusInfo()`, and in that case `authUser` is still being constructed so it should be OK to modify it rather than copying it.